api/keto/README.md

```diff
@@ -0,0 +1,31 @@
+# Notes
+
+> ORY Keto is still a `sandbox` project.
+This makes the included api version `v1` subject
```

Comment:
Okay, I wasn't sure whether we release next gen keto with a stable v1 directly, but this is not how it goes, since users will be using the service very early I guess.

In protobuf we can use `v1alpha1`, the first alpha of v1, then `v1beta1` and then eventually `v1` for stable definitions.

If we go that path we will always need to write version mappings of the messages versions on server side (map `v1alpha1` -> `v1beta1` -> `v1`), deprecate old versions and eventually remove the support at some point.

Sure, we can do this!

Reply:
Wouldn't it be possible to go with `v0` that can have breaking changes and then move on to `v1` once we release Keto v1.x.x? Otherwise, `v1alpha` and `v1beta` are good :+1:

Reply:
I would remove support for old versions immediately and guarantee compatibility only at a point where we are confident that not much will change.

Reply:
It depends on how long we have the alpha/beta versions public and how many clients are using an older version before should remove them. Generally the goal is to reach v1 as soon as possible.

**This is how versioning is usually seen for ProtoBuf APIs:**
- `v1alpha` - Highly subject to breaking changes, very early stage
- `v1beta` - Pretty reliable, may change. Should have a long evaluation time before going `v1`
- `v1` - Stable, API (RPCs, fields, messages) can only be added without breaking existing functionality

Reply:
Right, that's what I think as well. So we just start with `v1alpha` and move on when we think it is appropriate.